apps/web/src/components/layout/components/HeaderNav.tsx

```diff
@@ -85,8 +90,17 @@ export function HeaderNav({}: Props) {
           style={{ maxWidth: 150, maxHeight: 25 }}
         />
         <Group>
-          <ActionIcon variant="transparent" onClick={() => toggleColorScheme()} title="Toggle color scheme">
-            {dark ? <Sun {...headerIconsSettings} /> : <Moon {...headerIconsSettings} />}
+          <ActionIcon
+            variant="transparent"
+            onClick={() => toggleColorScheme()}
+            title={
```

Comment:
I think it's better to use the tooltip component from our design system? Or if there is no tooltip container there, we can use the mantine's default one

Reply:
Sure, I will look into it :)